### DISCUSSION TITLE
Import custom car 3D model

### DISCUSSION THREAD

**Discussion by mattbrth** [FS-Online/Driverless-Competition-Simulator#360] [General]
Hello, 
I need help to import a custom car 3D model.
My problem is that when start the simulation with a custom car 3D model, the car spawn and then the wheel turn and steer but the car doesn't move. I was careful to the box of the physical asset but still the car doesn't move.
Can someone share a FBX that works with the simulator so that a can compare what I did wrong.
Thank you !

**Comment by wouter-heerwegh** [comment]
Hi @MattBrth,

Are you sure the car is not spawned into the ground? If yiu hit backspace, it should reset and spawn you a little higher. Does that work?
